contracts/staking/contracts/src/interfaces/IStakingEvents.sol

```diff
@@ -134,4 +134,24 @@ interface IStakingEvents {
     event StakingPoolRewardVaultChanged(
         address rewardVaultAddress
     );
+
+    /// @dev Emitted when a staking pool's operator share is decreased.
+    /// @param poolId Unique Id of pool.
+    /// @param oldOperatorShare Previous share of rewards owned by operator.
+    /// @param newOperatorShare Newly decreased share of rewards owned by operator.
+    event OperatorShareDecreased(
+        bytes32 poolId,
+        uint32 oldOperatorShare,
+        uint32 newOperatorShare
+    );
+
+    /// @dev Emitted when an operator reward is transferred to the Eth vault.
+    /// @param amount The amount in ETH withdrawn.
+    /// @param operator of the pool.
+    /// @param poolId The pool the reward was deposited for.
+    event OperatorRewardTransferredToEthVault(
```

Comment:
Do we need this event? We already log all of this information on any deposit into the EthVault.

Reply:
Yeah, I see what you mean. Basically, the goal was to document the flow of value as it enters and leaves a contract. That said, we can probably nix now that the vaults are set during initialization (and they also emit both ingress/egress value flow).

Reply:
(Removed)